code/src/components/AddTask/AddTask.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import tasks from '../../reducers/tasks';
+import './AddTask.css';
+
+const AddTask = () => {
+  const [input, setInput] = useState('');
+  const dispatch = useDispatch();
+  const onAddTask = () => {
+    dispatch(tasks.actions.addTask(input));
+  };
+  const cleanInput = () => {
+    setInput('');
+  };
+
+  return (
+    <div className='form-container'>
+      {/* A form to control form elements and then pass the values into actions
+      after submitting. */}
+      <span className='input-span'>
+        <button
+          className='task-button'
+          onClick={() => {
+            onAddTask();
+            cleanInput();
+          }}
+        >
+          +
+        </button>
+        <input
+          className='form-input'
+          type='text'
+          value={input}
+          placeholder={'Add a crazy task'}
+          onChange={(event) => setInput(event.target.value)}
+        />
```

Comment:
A suggestion is to nest the input in a form-tag, in order to be able to submit just by pressing the enter key.

Reply:
I finally had the time to fix this. It works great! Thanks a lot for the tip :)